Reneiski_Homework1/ReneiskiSelenium11/PageObjects/LinksPage/LinksPage.cs

```diff
@@ -0,0 +1,32 @@
+﻿using OpenQA.Selenium;
+
+namespace ReneiskiSelenium11.PageObjects.LinksPage
+{
+    public class LinksPage : BasePage
+    {
+        public LinksPage(IWebDriver _driver) : base(_driver)
+        {
+        }
+
+        public string newUrl = "https://demoqa.com/";
+        public string noContent = "Link has responded with staus 204 and status text No Content";
+
+        private IWebElement currentUrl => _driver.FindElement(By.XPath("//a[@href='https://demoqa.com']"));
+
+        public string GetUrlAttribute() => currentUrl.GetAttribute("href");
+
+        public void SwitchPages(string value)
```

Comment:
I don't really like the fact that you need to pass a string a parameter to switch
I would better do 2 separate methods - one for switching first window, one for switching last window and do this into WebDriver Extension class so that call it just like WebDriverFactory.Driver.SwitchToTheLastWindow()

Reply:
Fixed